### PR TITLE
Refactor Slack plugin for readability.

### PR DESCRIPTION
**Pull request by tobli**
* Use lodash.get to simplify optional properties
* Sync metric titles for summary and attachments.

### CODE REVIEW

File: lib/plugins/slack/attachements.js [hunk 1]
```diff
@@ -12,89 +13,100 @@ function getMetric(metric) {
...
             short: false
           })
         }
       }
 
-      let color = metrics[options.limitMetric] ? metrics[options.limitMetric].metric > options.limitError ? 'good' : metrics[options.limitMetric].metric > options.limitWarning ? 'warning' : 'danger' : 'danger';
+      let color = 'danger';
 
-      attachements.push({
+      const limitMetric = metrics[options.limitMetric];
```
Comment (tobli) [inline lib/plugins/slack/attachements.js]: This looks strange to me. Shouldn't 'good' be the default? Also, the other checks seems backwards. For coachScore, higher is better, but for speedIndex and firstVisualChange the reverse is true.

File: lib/plugins/slack/summary.js [hunk 2]
```diff
@@ -1,75 +1,74 @@
...
+      name: 'Fully Loaded',
+      metric: get(base.browsertime, 'summary.fullyLoaded.median')
+    },
+    coachScore: {
+      name: 'Coach score',
+      metric: get(base.coach, 'summary.performance.score.median')
+    },
+    transferSize: {
+      name: 'Page transfer weight',
+      metric: h.size.format(get(base.pagexray, 'summary.transferSize.median'))
```
Comment (tobli) [inline lib/plugins/slack/summary.js]: This is a slight change that might not be desirable, formatting a missing transfer size as N/A.

**Comment by tobli** [comment]
Some small tweaks to reduce long lines and complex statements.

File: lib/plugins/slack/attachements.js (hunk 1 above)
Comment (soulgalore) [inline lib/plugins/slack/attachements.js]: yep this should be changed.

File: lib/plugins/slack/summary.js (hunk 2 above)
Comment (soulgalore) [inline lib/plugins/slack/summary.js]: I think that's ok for now

**Comment by tobli** [comment]
Cool. Just some drive-by changes to make it easier to make tweaks later.

**Pull request merged**